com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -0,0 +1,96 @@
+# High Definition Range (HDR) Output in HDRP
+
+With HDRP it is possible to output to High Dynamic Range (HDR) devices that allows for wider color gamut and higher brightness range.
+
+HDR Output can be enabled for a given platform in **Player Settings > Other Settings > Use display in HDR mode**.  When this option is enabled, HDRP will process the frame so that when an HDR screen is used as output device it is presented correctly.
+
+## Tonemapping for HDR Output
```

Comment:
This title is a bit confusing, because the tonemapping page seems to be talking about remapping HDR content to display on standard resolution displays. https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@13.1/manual/Post-Processing-Tonemapping.html

That page probably needs an update. 

Proposed alternative section title:
## Configuring Tonemapping settings for HDR displays